score.py

```diff
@@ -128,6 +134,109 @@ def run_inference(images_with_header: List[ImageWithHeader],
     return segmentation
 
 
+def extract_zipped_dicom_series(zip_file_path: Path, extraction_folder: Path) -> None:
```

Comment:
This is no longer specific to dicom, and could hence get a more generic name, that indicates that it flattens the folder structure?